components/gitpod-protocol/go/gitpod-service.go

```diff
@@ -2116,6 +2142,11 @@ type UpdateUserStorageResourceOptions struct {
 	URI     string `json:"uri,omitempty"`
 }
 
+// GetWorkspaceEnvVarsOptions is the GetWorkspaceEnvVarsOptions message type
+type GetWorkspaceEnvVarsOptions struct {
+	ProjectID string `json:"projectId,omitempty"`
```

Comment:
Why not parse WorkspaceID here?

Reply:
We can pass but a client still need to construct a scope which would allow to pass project guard?

Reply:
[Related](https://github.com/gitpod-io/gitpod/pull/16414#discussion_r1107078707)?